lldb/test/API/functionalities/progress_reporting/swift_progress_reporting/TestSwiftProgressReporting.py

```diff
@@ -38,14 +38,16 @@ def test_swift_progress_report(self):
             "Loading Swift module",
             "Importing modules used in expression",
             "Setting up Swift reflection",
-            "Getting Swift compile unit imports",
+            "Importing Swift module dependencies for main.swift",
```

Comment:
nit: We're only doing partial string matching on the test so we could have drop the file name at the end.